lib/Test/Mojo.pm

```diff
@@ -590,6 +621,36 @@ Access application with L<Mojo::UserAgent::Server/"app">.
   $t->get_ok('/hello')->status_is(200);
   is $stash->{foo}, 'bar', 'right value';
 
+=head2 attr_is
+
+  $t = $t->attr_is('img.cat' => 'alt', 'Grumpy cat');
+  $t = $t->attr_is('img.cat' => 'alt', 'Grumpy cat', 'right alt text');
```

Comment:
Considering there are 3 required args, the fat comma is probably not the right choice here.

Reply:
Replaced fat comma